cub/cub/device/dispatch/dispatch_unique_by_key.cuh

```diff
@@ -228,17 +270,42 @@ struct DispatchUniqueByKey
       }
 
       // Number of input tiles
-      constexpr auto block_threads    = VsmemHelperT::agent_policy_t::BLOCK_THREADS;
-      constexpr auto items_per_thread = VsmemHelperT::agent_policy_t::ITEMS_PER_THREAD;
-      int tile_size                   = block_threads * items_per_thread;
-      int num_tiles                   = static_cast<int>(::cuda::ceil_div(num_items, tile_size));
-      const auto vsmem_size           = num_tiles * VsmemHelperT::vsmem_per_block;
+      const auto block_threads = VSMemHelperT::template BlockThreads<
```

Comment:
Since `BlockThread` is a `consexptr` member function that returns `int`, use of `template` is unwarranted.